ffc/representation.py

```diff
@@ -192,6 +192,52 @@ def _compute_element_ir(ufl_element, element_numbers, classnames, parameters, ji
     return ir
 
 
+def _compute_dofmap_permutation_tables(fiat_element, cell):
+
+    if isinstance(fiat_element, MixedElement):
+        elems = fiat_element.elements()
```

Comment:
Is this correct? What about if a subelement is again a mixed element? What if it is other kind of compound element?

Reply:
I don't know. I don't have any examples at the moment, so I guess we can deal with it, when it comes up.